index.html

```diff
@@ -14,17 +14,21 @@
       src="https://kit.fontawesome.com/04970764d6.js"
       crossorigin="anonymous"
     ></script>
-    <link rel="stylesheet" media="(min-width: 992px)" href="media.css">
-    <link rel="stylesheet" media="(max-width: 992px)" href="style.css">
+    <link rel="stylesheet" media="(min-width: 992px)" href="desktop.css">
+    <link rel="stylesheet" media="(max-width: 992px)" href="mobile.css">
   </head>
   <body>
     <header class="header">
-      <a href="#" class="logo">My Logo</a>
-      <nav class="menu"><i class="fas fa-bars"></i></nav>
+      <a href="#" class="logo">My Logo</a>      
+      <div class="menu">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
```

Comment:
## GOOD JOB 👏

- [ ] Everything is fixed as previous reviewer suggested 😃. There just on little thing that you should fix 🔨. Let's remove the padding from the header or `.header` selector. I am sure you can fix it easily 😎

**See the image below where to see should be removed** 
![image](https://user-images.githubusercontent.com/69011963/151935510-861d4b73-1b4e-4114-a80a-5120fba4a25b.png)